fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/PlanStringRepresentationTest.java

```diff
@@ -497,7 +497,17 @@ void doNotEvaluateOutsideOfLimit() {
         Random r = ThreadLocalRandom.current();
         List<RecordQueryPlan> plans = new ArrayList<>();
         for (int i = 0; i < 3; i++) {
-            plans.add(randomPlanAndString(r).getLeft());
+            RecordQueryPlan plan;
+            while (true) {
+                plan = randomPlanAndString(r).getLeft();
+                try {
+                    plan.isReverse();
+                    break;
+                } catch (RecordCoreException e) {
+                    // Not all plans have a defined reverse-ness. If this happens, generate a new plan
```

Comment:
This seems like we might be missing something here, but I think it should be covered by the other tests